multipath/custom.conf

```diff
@@ -0,0 +1,6 @@
+# Custom configuration for multipathd
+
+# Changes made to this file will not be overwritten by future system updates.
+# They will also be retained through system upgrades to newer releases.
```

Comment:
This is not true at the moment, but I suggest we make it become true. If this is seen as a good idea, we can probably contribute the change to the installer, unless there are important technical challenges.